lib/galaxy/tool_util/deps/container_classes.py

```diff
@@ -83,6 +83,19 @@ def resolve_dependencies(self):
     def shell(self):
         return DEFAULT_CONTAINER_SHELL if not self.container_description else self.container_description.shell
 
+    @property
+    def source_environment(self):
+        return """if [ -d /usr/local/etc/conda/activate.d ]; then
```

Comment:
Do we want to add a comment into the script, that this is for automatically build containers?

Reply:
Sure, I can do that